src/car/smart/SmartCar.hpp

```diff
@@ -87,7 +87,7 @@ class SmartCar : public DistanceCar, public HeadingCar
      * }
      * \endcode
      */
-    void update() override;
+    virtual void update() override;
```

Comment:
:warning: **modernize-use-override** :warning:
'virtual' is redundant since the function is already declared 'override'
```suggestion
    virtual void ```